### DISCUSSION TITLE
Platform.is.installedPWA implementation suggestion

### DISCUSSION THREAD

**Discussion by FKPSC** [quasarframework/quasar#15362] [CLI - PWA mode]
It would be great (and hopefully easy) to have `Platform.is.installedPWA`  to easily check if the current platform is a browser tab or a standalone installed PWA.

**Comment by pdanpdan** [comment]
https://web.dev/learn/pwa/detection/#detecting-the-transfer

**Comment by FKPSC** [comment]
I know there are ways to detect, not only PWAs, but all of the platforms that are included in `Platform.is.*`.
The idea is to add `PWA` as distinct platform from a browser tab environment.

Quasar treats PWA as an additional mode on top of SPA. Since there is platform detection for every other Quasar mode, it would be nice to complete the set.

If it is particularly challenging and needs to be handled by each developer in a specific way, it would be really useful to include a tip/note/warning on the platform page about what developers need to watch out for in this situation.
